terraform/dev3.tfvars

```diff
@@ -0,0 +1,19 @@
+meshdb_proxmox_node             = "jon"
+meshdb_proxmox_template_image   = "debian-cloud"
+meshdb_proxmox_storage_location = "local-lvm"
+meshdb_env_name                 = "dev3"
+meshdb_local_user               = "debian"
+meshdb_mgr_ips = [
+  "10.70.90.146",
+]
+meshdb_ips = [
+  "10.70.90.147",
+  "10.70.90.143",
+  "10.70.90.144",
+]
+meshdb_lb_ip        = "10.70.90.145"
+meshdb_gateway      = "10.70.90.1"
+meshdb_networkrange = "24"
+meshdb_net_block    = "10.70.90.0"
+meshdb_external_ip  = "199.170.132.46"
```

Comment:
Woa, two public IPs (one for dev, one for prod), high roller! We better make good use of these 😤